exercises/practice/isbn-verifier/example.h

```diff
@@ -0,0 +1,10 @@
+#if !defined(EXAMPLE_H)
+#define EXAMPLE_H
```

Comment:
I think in the other exercises we use the name of the exercise for the include guard, so `ISBN_VERIFIER_H` instead of `EXAMPLE_H`